stdlib/list.mli

```diff
@@ -57,6 +57,12 @@ val compare_length_with : 'a list -> int -> int
    @since 4.05.0
  *)
 
+val is_empty : 'a list -> bool
+(** [is_empty l] is true if and only if [l] has no elements. It is equivalent to
+    [compare_length_with l 0 = 0].
+    @since 5.00.0
```

Comment:
The `@since` annotation needs to be changed to `5.1`.

Reply:
Done by direct push - this can all be squashed when CI comes back

Reply:
ListLabels is missing ! :)

Reply:
🤦